recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.8.17" %}
+{% set version = "1.8.18" %}
 
 package:
   name: hdf5
   version: {{ version }}
 
 source:
   fn: hdf5-{{version}}.tar.gz
-  url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-{{ version }}/src/hdf5-{{ version }}.tar.gz
-  md5: 7d572f8f3b798a628b8245af0391a0ca
+  url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-{{ version }}/src/hdf5-{{ version }}.tar.gz
```

Comment:
Have addressed this in `master`. So will need another rebase for this PR to clean this stuff out.